run_integration.sh

```diff
@@ -8,7 +8,7 @@ pushd $rootdir
 # Reset test harness
 rm -rf test-harness
 # TODO: Before merging, change branch back to master
-git clone --single-branch --branch feature/box-storage https://github.com/algorand/algorand-sdk-testing.git test-harness
+git clone --single-branch --branch indexer-boxes https://github.com/algorand/algorand-sdk-testing.git test-harness
```

Comment:
Same issue as with the other 2. This file shouldn't exist.

Reply:
correct, now we are changing inside of `.test-env`